src/app/components/ElectronicDelivery/ElectronicDelivery.jsx

```diff
@@ -52,11 +52,16 @@ class ElectronicDelivery extends React.Component {
 
   /*
    * componentDidUpdate()
-   * After the component updates, if there are errors then the DOM for the error box message
+   * If the component makes a request, it will focus on the loading layer component.
+   * Also, after the component updates, if there are errors then the DOM for the error box message
    * is rendered. Since it exists, it should be focused so that the patron can get a better
    * idea of what is wrong.
    */
   componentDidUpdate() {
+    if (this.loadingLayer) {
+      this.loadingLayer.focus();
```

Comment:
So this doesn't seem to be working, right?

Reply:
I tested it locally and on dev on my machine, I do see it works, but seems not work on other places. It must be something off. I am working on investigating it.